superset-frontend/src/views/CRUD/chart/ChartList.tsx

```diff
@@ -325,13 +327,20 @@ function ChartList(props: ChartListProps) {
               changed_by_url: changedByUrl,
             },
           },
-        }: any) => (
-          <a href={changedByUrl}>
-            {lastSavedBy?.first_name
-              ? `${lastSavedBy?.first_name} ${lastSavedBy?.last_name}`
-              : null}
-          </a>
-        ),
+        }: any) =>
+          enableBroadUserAccess ? (
+            <a href={changedByUrl}>
+              {lastSavedBy?.first_name
```

Comment:
A minor enhancement, but since this is the same as the [one below](https://github.com/apache/superset/pull/21302/files#diff-d7e43cf67304799e8b43fc7da50111f67cbd144ea479b71a9f9d3207a417fdd9R334), can you please put it in a separate constant and just reference it in both places